### ISSUE TITLE
Supporting files in messages

### ISSUE THREAD

**Issue by ansman** [alexkvak/teamcity-slack#108]
We generate a release notes file that would be great to include either as an attachment or as text in the message.

I would be happy to take a stab at integrating this if need be.

**Comment by alexkvak** [comment]
Hi,

you can attach link to the artifact

**Comment by ansman** [comment]
Right. I suppose I could make the release notes an artifact but it would be nice if that wasn't a requirement.

**Comment by ansman** [comment]
Oh, sorry. I don't want a link to the artifact. I want to attach it to the slack message as an attachment or include the contents in the message

**Comment by alexkvak** [comment]
There are so many requests for the new template fields, I guess we need plugin system for this plugin :))

**Comment by ansman** [comment]
I was thinking more that it could be a special syntax in the message that is handled, maybe something like: `{attachment:%someFile%}` or `{contents:%someFile%}`
